cpp/ps0000/p0Solution.cpp

```diff
@@ -1,13 +1,17 @@
-#include<iostream> /*"iostream" not "iostream.h" is the correct header file to include for using input-output 
-stream objects in new verions of c++ */
+#include <iostream> // 2/2
+/*"iostream" not "iostream.h" is the correct header file to include 
+for using input-output stream objects in new verions of c++ */

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`